benchmark_runner/workloads/workloads_operations.py

```diff
@@ -447,7 +449,9 @@ def clear_nodes_cache(self):
         """
         This method clear nodes cache
         """
-        self._oc.clear_node_caches()
+        for i in range(self.REPEAT_TIMES):
+            self._oc.clear_node_caches()
+            time.sleep(self.SLEEP_TIME)
```

Comment:
1. This should only sleep after the first N-1 repeat times.
2. 30 seconds between drop caches is excessive (90 seconds, or 60 when you fix (1)).  3 seconds should be fine.

Reply:
1. I dont see any problem to do it after each cleaning, if it will not solve the issue we are going to revert it back.
2. I will change it to 3 sec